PrimateAI.pm

```diff
@@ -0,0 +1,160 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2021] EMBL-European Bioinformatics Institute
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+     http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+  Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+  PrimateAI
+
+=head1 SYNOPSIS
+
+  mv PrimateAI.pm ~/.vep/Plugins
+
+  ./vep -i variations.vcf --plugin PrimateAI,PrimateAI_scores_v0.2.tsv.bgz
+  ./vep -i variations.vcf --plugin PrimateAI,PrimateAI_scores_v0.2_hg38.tsv.bgz
+
+=head1 DESCRIPTION
+
+  The PrimateAI VEP plugin is designed to retrieve clinical impact scores of variants, as described in https://www.nature.com/articles/s41588-018-0167-z. In brief, common missense mutations in non-human primate species are usually benign in humans. They can therefore be eliminated from screens for pathogenic mutations.
+
+  This plugin uses files generated by the PrimateAI software, which is available from https://github.com/Illumina/PrimateAI. The files containing predicted pathogenicity scores can be downloaded from https://basespace.illumina.com/s/yYGFdGih1rXL (a free BaseSpace account may be required):
+      PrimateAI_scores_v0.2.tsv.gz (for GRCh37/hg19)
+      PrimateAI_scores_v0.2_hg38.tsv.gz (for GRCh38/hg38)
+
+  Before running the plugin for the first time, the following steps must be taken to format the downloaded files:
+  1.  Unzip the score files
+  2.  Add '#' in front of the column description line
+  3.  Remove any empty lines.
+  4.  Sort the file by chromosome and position
+  5.  Compress the file in .bgz format
+  6.  Create tabix index (requires tabix to be installed).
+  Command line example:
```

Comment:
Suggest adding an empty line here for readability and maybe using ">" instead of $